_posts/ericabrody/2016-01-012-Erica's-input_turtle- assign.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: ericabrody
+title: "Erica's First Exercise"
+---
+
+# Here is the raw_input program:
+<iframe src="https://trinket.io/embed/python/b62728b39e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+# Here is the turtle hacking program:
+<iframe src="https://trinket.io/embed/python/9f350436bc" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflections
+I got stuck on the raw_input program initially, because it took me a while to figure out that when the color was a variable, it 
+didn't need quotes around it.
+
+For the turtle program, I wanted to play with color. Initially, I tried to create a small square and fill it in and then make a bigger 
+square on top. However, the fill color of the bigger square covered up the small square. So I figured out that it would be better to draw 
+the biggest square first and then draw smaller squares on top of it. One remaining question, I couldn't figure out why the edges
+of the squares aren't showing when the square is first drawn, we just see the fill. 
```

Comment:
Great question.  Check the state of your turtle's pen.  Your turtle will only draw while 1) moving and 2) with pen down.